vendor/github.com/gorilla/rpc/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2012 Rodrigo Moraes. All rights reserved.
```

Comment:
fwiw, I don't super care, but I'm trying to limit dependency sprawl from encompassing the whole internet.